model/change_point_detection.go

```diff
@@ -298,15 +328,11 @@ func clearChangePoints(ctx context.Context, env cedar.Environment, performanceRe
 	return errors.Wrap(err, "Unable to clear change points")
 }
 
-func createChangePoint(ctx context.Context, env cedar.Environment, resultToUpdate string, measurement string, algorithm AlgorithmInfo) error {
+func createChangePoint(ctx context.Context, env cedar.Environment, resultToUpdate string, cp ChangePoint) error {
```

Comment:
I'd leave this the way it was.

Reply:
We need to persist the triage section as well now; rather than keeping adding arguments to this, it seems simpler to just persist the change point itself IMO.

Reply:
right, but doing this forces us to leak our data layout into the implementation details of the change detector, which feels more complicated to maintain long term.

Reply:
Cleaned up the use of model package in the change detector as discussed.